diff --git a/makefile b/makefile
index e13a99b4..1b2c36cf 100644
--- a/makefile
+++ b/makefile
@@ -14,14 +14,14 @@
 # along with this program; if not, write to the Free Software Foundation,
 # Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
-DESTDIR = $(CURDIR)/
-PREFIX = CEdev
+DESTDIR = @out@
+PREFIX =
 
 include $(CURDIR)/src/common.mk
 
 LIBS := libload graphx fontlibc keypadc fileioc usbdrvce srldrvce msddrvce fatdrvce lcddrvce
 SRCS := ce crt libc libcxx softfloat
-TOOLS := fasmg convbin convimg convfont cedev-config cedev-obj
+TOOLS := fasmg cedev-config cedev-obj
 
 
 ifeq ($(OS),Windows_NT)
@@ -56,9 +56,6 @@ install: all $(addprefix install-,$(SRCS)) $(addprefix install-,$(LIBS))
 	$(Q)$(call COPYDIR,$(call NATIVEPATH,examples/*),$(INSTALL_EXAMPLES))
 	$(Q)$(call COPY,$(call NATIVEPATH,src/makefile.mk),$(INSTALL_META))
 	$(Q)$(call COPY,$(call NATIVEPATH,src/linker_script.ld),$(INSTALL_META))
-	$(Q)$(call COPY,$(call NATIVEEXE,tools/convfont/convfont),$(INSTALL_BIN))
-	$(Q)$(call COPY,$(call NATIVEEXE,tools/convimg/bin/convimg),$(INSTALL_BIN))
-	$(Q)$(call COPY,$(call NATIVEEXE,tools/convbin/bin/convbin),$(INSTALL_BIN))
 	$(Q)$(call COPY,$(call NATIVEEXE,tools/cedev-config/bin/cedev-config),$(INSTALL_BIN))
 	$(Q)$(call COPY,$(call NATIVEEXE,tools/cedev-obj/bin/cedev-obj),$(INSTALL_BIN))
 	$(Q)$(WINDOWS_COPY)
@@ -70,7 +67,7 @@ $(addprefix install-,$(LIBS)): $(TOOLS)
 	$(Q)$(MAKE) -C $(call SRCDIR,$(patsubst install-%,%,$@)) install PREFIX=$(PREFIX) DESTDIR=$(DESTDIR)
 
 libs: $(LIBS) $(TOOLS)
-	$(Q)$(call NATIVEEXE,tools/convbin/bin/convbin) --oformat 8xg-auto-extract \
+	$(Q)$(call NATIVEEXE,convbin) --oformat 8xg-auto-extract \
 		$(foreach library,$(LIBS),$(addprefix --input ,$(call SRCDIR,$(library))/$(library).8xv)) --output clibs.8xg
 
 libs-zip:
diff --git a/src/makefile.mk b/src/makefile.mk
index 76e56265..a53c81b3 100644
--- a/src/makefile.mk
+++ b/src/makefile.mk
@@ -51,7 +51,7 @@ PREFER_CE_LIBC ?= YES
 SKIP_LIBRARY_LDFLAGS ?= NO
 LIBLOAD_OPTIONAL ?=
 COMPRESSED_MODE ?= zx7
-COMMENT ?= $(shell cedev-config --comment)
+COMMENT ?=
 #----------------------------
 CEDEV_TOOLCHAIN ?= $(shell cedev-config --prefix)
 #----------------------------
diff --git a/tools/cedev-config/Makefile b/tools/cedev-config/Makefile
index 7d557f79..33dc8e59 100644
--- a/tools/cedev-config/Makefile
+++ b/tools/cedev-config/Makefile
@@ -32,8 +32,7 @@ else
     LDFLAGS += -mmacosx-version-min=10.13
   else
     STRIP = strip --strip-all "$1"
-    CFLAGS += -D_XOPEN_SOURCE=500 -static
-    LDFLAGS += -static
+    CFLAGS += -D_XOPEN_SOURCE=500
     LIBRARIES += dl
   endif
 endif
diff --git a/tools/cedev-obj/Makefile b/tools/cedev-obj/Makefile
index 16030d45..98f986c3 100644
--- a/tools/cedev-obj/Makefile
+++ b/tools/cedev-obj/Makefile
@@ -32,8 +32,7 @@ else
     LDFLAGS += -mmacosx-version-min=10.13
   else
     STRIP = strip --strip-all "$1"
-    CFLAGS += -D_XOPEN_SOURCE=500 -static
-    LDFLAGS += -static
+    CFLAGS += -D_XOPEN_SOURCE=500
     LIBRARIES += dl
   endif
 endif
